docs/UserGuide/Data-Concept/Encoding.md

```diff
@@ -54,6 +54,12 @@ Usage restrictions: When using GORILLA to encode INT32 data, you need to ensure
 
 DICTIONARY encoding is lossless. It is suitable for TEXT data with low cardinality (i.e. low number of distinct values). It is not recommended to use it for high-cardinality data. 
 
+* FREQ
+
+FREQ encoding is lossy. It transforms the time sequence to the frequency domain and only reserve part of the frequency components with high energy. It is more suitable for sequence with obvious periodicity.
+
+> There are two parameters of FREQ encoding in the configuration file: `freq_snr` defines the signal-noise-ratio (SNR), `freq_block_size` defines the data size in a time-frequency transformation.
```

Comment:
How do these two parameters affect the encoding performance and compression ratio?

Reply:
The experimental results and analysis have been put on Apache confluence https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=199536917

Reply:
Yes, I mean, to tell users how to use these parameters, we can show the conclusion in this user guide here. :)

Reply:
Yes, I mean, to tell users how to use these parameters, we can show the conclusion in this user guide here. :)

Reply:
The brief conclusion is added in the user guide.